### ISSUE TITLE
Using SemVer throws a fatal error

### ISSUE THREAD

**Issue by roland-d** [consolidation/Robo#510]
### Steps to reproduce
1. I created a new RoboFile using ```robo init```
2. I added the following code:
```
public function hello($world)
	{
		// Load the semver handler
		$this->taskSemVer('.semver')
			->increment()
			->run();

		$this->say("Hello, $world");
	}
```
3. Ran the script as ```robo hello world```

### Expected behavior
I would expect to see ```Hello, world```

### Actual behavior
What I see is this:
$ robo hello world

Fatal error: Uncaught Error: Call to undefined method Robo\Task\Development\SemVer::inflect() in C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\Collection\CollectionBuilder.php on line 369

Error: Call to undefined method Robo\Task\Development\SemVer::inflect() in C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\Collection\CollectionBuilder.php on line 369

Call Stack:
    0.0002     364840   1. {main}() C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\robo:0
    0.0042     692224   2. Robo\Runner->execute() C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\robo:20
    0.0137    1290152   3. Robo\Runner->run() C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\Runner.php:111
    0.3237    4476824   4. Symfony\Component\Console\Application->run() C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\Runner.php:157
    0.3411    4477656   5. Symfony\Component\Console\Application->doRun() C:\Users\User\AppData\Roaming\Composer\vendor\symfony\console\Application.php:122
    0.3413    4477656   6. Symfony\Component\Console\Application->doRunCommand() C:\Users\User\AppData\Roaming\Composer\vendor\symfony\console\Application.php:191

ERROR: Uncaught Error: Call to undefined method Robo\Task\Development\SemVer::inflect() in C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\Collection\CollectionBuilder.php:369
Stack trace:
#0 C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\Collection\CollectionBuilder.php(357): Robo\Collection\CollectionBuilder->fixTask(Object(Robo\Task\Development\SemVer), Array)
#1 C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\TaskAccessor.php(45): Robo\Collection\CollectionBuilder->build('Robo\\Task\\Devel...', Array)
#2 C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\Task\Development\loadTasks.php(44): Robo\Tasks->task('Robo\\Task\\Devel...', '.semver')
#3 D:\wamp64\www\root\deploy\RoboFile.php(24): Robo\Tasks->taskSemVer('.semver')
#4 [internal function]: RoboFile->hello('world', Array)
#5 C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\annotated-command\src\CommandProcessor.php(180): call_user_func_array(Array, Array)
#6
in C:\Users\User\AppData\Roaming\Composer\vendor\consolidation\robo\src\Collection\CollectionBuilder.php:369

### System Configuration
Which O.S. and PHP version are you using?
Using Windows 10 and PHP 7.0.10

### Additional information
This script is works when I use codegyre/robo but this gave me a warning saying "Package codegyre/robo is abandoned, you should avoid using it. Use consolidation/robo instead."

For now I have reverted but just wanted to let you know about the error.

**Comment by luckyshot** [comment]
Same happening here:
`Fatal error: Call to undefined method Robo\Task\Development\SemVer::inflect() in /.../vendor/consolidation/robo/src/Collection/CollectionBuilder.php on line 369
ERROR: Call to undefined method Robo\Task\Development\SemVer::inflect()
in /.../web/vendor/consolidation/robo/src/Collection/CollectionBuilder.php:369`

**System Configuration**

- Mac OSX 10.11.6 El Capitan
- PHP 5.6.10
- consolidation/robo 1.0.5

I tried to fix it myself but to no avail, rolling back to `codegyre/robo`

**Comment by cyberhorse** [comment]
Same here

**Comment by greg-1-anderson** [comment]
Fixed in #528. I will tag a 1.0.6 shortly.
